browser/src/slideshow/engine/SlideAnimations.ts

```diff
@@ -42,6 +42,7 @@ function isAttributeSupported(aNodeInfo: AnimateNodeInfo) {
 		console.error('slideshow: missing attributeName');
 		return false;
 	}
+	// return true;
```

Comment:
leftover? please remove later

Reply:
it's just useful when you need to relax constrains for developing support for more animations
surely it has to be removed on the long term